driver/src/order_driver.rs

```diff
@@ -47,6 +47,11 @@ pub fn run_order_listener<D, C>(
             info!("Standing Orders: {:?}", standing_orders);
             info!("All Orders: {:?}", orders);
 
+            let standing_order_indexes_u32 = get_standing_orders_indexes(&standing_orders);
```

Comment:
Why does `get_standing_orders_indexes` not return `Vec<u128>`? Also I believe you should be able to convert a u32 into a u128 by just calling `x as u128`. No need to convert into a string and back.